android/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java

```diff
@@ -173,10 +173,11 @@ public WebResourceResponse shouldInterceptRequest(WebResourceRequest request) {
       return null;
     }
 
-    if (isLocalFile(loadingUrl) || loadingUrl.toString().startsWith(bridge.getLocalUrl())) {
+    if (isLocalFile(loadingUrl) || !bridge.getAppUrl().contains("localhost:8100") && loadingUrl.toString().startsWith(bridge.getLocalUrl())) {
```

Comment:
this is not a good check, will stop working if the user changes the default port. It's probably better to just check `Config.getString("server.url") == null`